### DISCUSSION TITLE
Configure self-hosted domain on the desktop client

### DISCUSSION THREAD

**Discussion by Mekacher-Anis** [outline/outline#6432] [Self Hosting]
I setup a local self-hosted instance for testing and so far it works great. Awesome Job!!
My question is how can I setup my custom domain on the MAC desktop client to be able to login to myself hosted instance?

**Comment by tommoor** [comment]
Hi @Mekacher-Anis – the desktop app is only available for cloud-hosted customers at this time, perhaps in the future

**Reply by Mekacher-Anis** [reply]
can I help implement it?

**Reply by tommoor** [reply]
Hi, the desktop app codebase is private at the moment.
